modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
+Displays the Runtime Fabric version information
 
-.. Make sure to base64 encode the license, following the instructions in xref:install-manual#base64-encode-your-mule-license-key[Base64 Encode your Mule License Key].
-.. Run the following command:
+----
+rtfctl version
+----
+
+== Commands for Managing Applications on Runtime Fabric
+
+Use the following commands to manage applications deployed to Runtime Fabric.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Command |Description
+|xref:delete[`delete`] | Deletes a secure property
+|xref:describe[`describe`] | Displays details for an application
+|xref:disk[`disk`] | Displays disk usage for an application
+|xref:memory[`memory`] | Displays the JVM memory usage in a pod
+|xref:package[`package`] | Creates a debugging information package for an application
+|xref:restart[`restart`] | Restarts an application
+|===
+
+=== `delete`
+Deletes a secure property
+----
+rtfctl delete secure-property <sample-key> -n <sample-app-namespace>
+----
+
+=== `describe`
+Displays details for an application in a specific namespace 
+
+----
+rtfctl describe app <app-name> --namespace <namespace>
+----
+
+=== `disk`
+
+Displays disk usage for an application
+
+* Displays the disk usage for an application, using the first replica by default.
++
+----
+rtfctl disk <app-name>
+----
+
+* Displays the disk usage for an application, using a specific replica
++
+----
+rtfctl disk <app-name> --details --pod <replica-name>
+----
+
+=== `memory`
+Displays the JVM memory usage in a pod
+
+* Gets the JVM memory for each replica
++
+----
+rtfctl memory <app-name>
+----
+
+* Gets the JVM memory, for a particular replica
++
+----
+rtfctl memory <app-name> --pod <app-name>-pod-1
+----
+
+=== `package` 
+Creates a debugging information package for an application
+
+* Creates a package for an application, using the first pod by default
 +
 ----
-$ sudo ./rtfctl apply mule-license '<license-key-information>'
+rtfctl package <app-name>
+----
+
+* Created a package for an application containing the apps, policies, and `.mule` directories
++
+----
+rtfctl package <app-name> --apps --policies --dotmule
+----
+
+* Creates a package for an application containing a heap dump
++
+----
+rtfctl package <app-name> --heap-dump
+----
+
+=== `restart`
+Restarts an application
+
+----
+rtfctl restart my-app
+----
+
+
+== Commands for Troubleshooting Runtime Fabric
+
+Use the following commands to troubleshoot Runtime Fabric.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Command |Description
+|xref:heapdump[`heapdump`] | Triggers a JVM heap dump
+|xref:threaddump[`threaddump`] | Triggers a JVM thread dump
+|xref:get[`get`] | Displays one or more resources
+|xref:ping[`ping`] | Tests network connectivity
+|xref:report[`report`] | Generates a diagnostics report
+|xref:status[`status`] | Displays the status of Runtime Fabric
+|xref:test[`test`] | Verifies outbound network connectivity to the Runtime Fabric control plane
+|===
+
+=== `heapdump`
```

Comment:
I would put heapdump and threaddump in the application section - they are troubleshooting commands, but specific to applications, not the Runtime Fabric cluster itself

Reply:
ok, moved